packages/resolve-redux/src/set_subscription_middleware.js

```diff
@@ -3,14 +3,14 @@ import socketIOClient from 'socket.io-client';
 
 const CRITICAL_LEVEL = 100;
 
-function createMiddleware({ rootDirPath }) {
+function createMiddleware() {
     return (store) => {
         let socketIOFailCount = 0;
         let socketIO = null;
 
         const initSocketIO = () => {
             socketIO = socketIOClient(window.location.origin, {
-                path: `${rootDirPath}/socket/`
+                path: `${process.env.ROOT_DIR}/socket/`
```

Comment:
This will not work since following reason: it's client-side code, and `process.env` variable is not real - it had been substituted by webpack, when in bundled in client code. Since code above is library's code, it will not be processed by webpack. So after this PR you just will have in browser error like `process is null or undefined`